packages/core/botpress/src/web/components/App/index.jsx

```diff
@@ -17,6 +18,9 @@ import {
 } from '~/actions'
 
 class App extends Component {
+  toastNLUId = null
```

Comment:
We can't introduce module-specific logic inside the core

Reply:
@slvnperron I've tried implement is separate from core but meet one trouble `toast` will disappear if you go out from `QnA module`. Is it normal?

Reply:
@dmytropanontko , I think it's not fine, since an idea here is to notify user if something went wrong in the background. So I think the solution is to make an API more generic so that you can reuse it in other modules.

Reply:
@epaminond okey, I will made it tommorow